universe/auto_syncer.go

```diff
@@ -109,7 +110,13 @@ func (f *FederationEnvoy) Start() error {
 		})
 		err := f.AddServer(serverAddrs...)
 		if err != nil {
-			log.Warnf("unable to add universe servers: %v", err)
+			// On restart, we'll get an error for universes already
```

Comment:
I think we want to actually check against the expected error here. Instead of re-wrapping to add that text context, we want to just ignore it all together and move on (no need to log as it'll be printed each time).

Reply:
Agree, though I hit a dependency loop when trying to check against an error defined in tapdb/sqlerrors. Will look at defining it elsewhere.

Reply:
Agree, though I hit a dependency loop when trying to check against an error defined in tapdb/sqlerrors. Will look at defining it elsewhere.

Reply:
Yeah I think we should define it in the `universe` package, then the error is returned like in some other similar areas.